packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -129,6 +128,39 @@ class FocusAttachment {
   }
 }
 
+/// Describe when should happen after [FocusNode.unfocus] is called.
+///
+/// See also:
+///
+///  * [FocusNode.unfocus], which takes this as its `disposition` parameter.
+enum UnfocusDisposition {
+  /// Focus the [FocusManager.rootScope]. This is often the least useful thing
+  /// to do with focus, since the root scope is effectively above any node that
+  /// appears in the widget tree, there is no context for it, and thus no
+  /// FocusTraversalPolicy that can be used to determine a next focus. This
+  /// means that if you hit TAB, you go nowhere.
+  ///
+  /// But, if you need a place to park to focus which will always be valid, this
+  /// is the option to use. It is most frequently used when an explicit settgin
+  /// of the focus will follow.
+  root,
+
+  /// Focus the nearest focusable enclosing scope of this node without trying to
```

Comment:
the docs for "scope" and "pop" are very technical... I wish they'd provide a more intuitive understanding of what happens to the focus...

Reply:
In other words: Based on the docs here I don't really know which one I should be passing to `unfocus`.